### DISCUSSION TITLE
Disable button drop effect on click

### DISCUSSION THREAD

**Discussion by najibghadri** [mantinedev/mantine#4252] [Q&A] [answered]
Hello! Is there a way to disable the drop effect on all clickables in Mantine?

**Comment by rtivital** [comment]
What is drop effect?

**Reply by najibghadri** [reply]
https://github.com/mantinedev/mantine/assets/11639734/425a7db2-60af-47c0-a988-84ba7d56d036

@rtivital 
Not sure how to call this effect other than drop. When it's clicked it drops/translates downward. Can it be disabled globally somehow? All clickables have this even the buttons in calendar for example. In our ux it doesn't really fit unfortunately

**Reply by rtivital** [reply, marked as answer]
https://mantine.dev/theming/theme-object/#activestyles

**Reply by najibghadri** [reply]
Oh wow, I missed this, thank you!

**Reply by najibghadri** [reply]
Thank you for your contribution to this library btw @rtivital. I think on of the most clean and complete UI lib out there.
